packages/terra-form-select/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Changelog
 
 ## Unreleased
-
+* Added
+  * Added aria-required property to avoid displaying browser default tooltip.
```

Comment:
Let's give this some breathing room :)
```suggestion

* Added
  * Added aria-required property to avoid displaying browser default tooltip.
  
```

Reply:
@AH106586Harika this change log is missing details about aria-label changes please include.

Reply:
Added aria-label details on  CHANGELOG